Alert.css

```diff
@@ -0,0 +1,71 @@
+.header {
+  text-align: center;
+  color: indianred;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.alert {
+  border: 2px solid black;
+  height: 20px;
+  padding: 15px 10px;
+  width: 100%;
+  border-radius: 0.4rem;
```

Comment:
generally, we use rem for values greater than 10px
here you can use 4px for border-radius instead of 0.4rem

Reply:
Thanks Dheeraj